version/1/4/heb/code_of_conduct.md

```diff
@@ -0,0 +1,65 @@
+# קוד התנהגות לתורם
+
+## ההתחייבות שלנו
+
+לטובת עידוד של סביבה פתוחה ומקבלת, אנחנו בתור התורמים והמתחזקים מתחייבים להפוך 
+את ההשתתפות בפרויקט שלנו ואת הקהילה שלנו לחוויה נטולת הטרדות לכל אדם באשר הוא
+ללא הבדלי גיל, משקל, מגבלות פיזיות, מוצא אתני, הזדהות מגדרית, רמת ניסיון, לאום, 
+מראה חיצוני, גזע, דת, זהות ונטיה מינית.
+
+## הסטנדרטים שלנו
+
+דוגמאות של התנהגות התורמת לסביבה חיובית כוללות:
+
+* שימוש בשפה מקבלת ומברכת
+* כיבוד של נקודות מבט וחוויות שונות
+* קבלה של ביקורת בהבנה
+* התמקדות במה שהכי טוב לקהילה
+* הפגנת אמפתיה כלפי חברים אחרים מהקהילה
+
+דוגמאות של התנהגות לא מקובלת כוללות:
+
+* שימוש בשפה מינית, תמונות גסות, או הטרדה מינית
+* הטרלה, תגובות מעליבות/משפילות, תקיפה אישית או פוליטית
+* הטרדה פומבית או אישית
+* פרסום מידע אישי של אחרים, כגון כתובת פיזית או אלקטרונית, בלי אישור מפורש
+* התנהגות כלשהי שאינה סבירה בסביבת עבודה מקצועית
+
+## האחריויות שלנו
+
+מתחזקי הפרויקט אחראיים להבהיר את הסטנדרטים להתנהגות מקובלת ומצופים לפעול באופן 
+מתאים והוגן בתגובה לכל אירוע של התנהגות בלתי מקובלת.
+
+יש למתחזקי הפרויקט את הזכות והאחריות להסיר, לערך, או לדחות תגובות, קומיטים, קוד, 
+עדכוני וויקי, נושאים, או תרומות אחרות שאינם מיושרות עם קוד ההתנהגות הנ״ל, או לחסום
```

Comment:
The GitHub "Issues" term translates poorly to `נושאים`. A better translation would be `סוגיות`.